force-app/main/default/aura/createServiceSchedule/createServiceSchedule.cmp

```diff
@@ -0,0 +1,11 @@
+<aura:component
+    implements="flexipage:availableForRecordHome, force:hasRecordId, lightning:actionOverride, lightning:isUrlAddressable"
+    access="global"
```

Comment:
@jjbennett does this one have to be global to be in the action override?

Reply:
99% sure in order for it to be seen in the choices of overrides.